src/Api/Users.php

```diff
@@ -553,6 +553,62 @@ public function removeImpersonationToken(int $user_id, int $impersonation_token_
         return $this->delete('users/'.self::encodePath($user_id).'/impersonation_tokens/'.self::encodePath($impersonation_token_id));
     }
 
+    /**
+     * @param int   $user_id
+     * @param array $params
```

Comment:
We need documentation on the values that go in here - I know not all the code has this, but all new code needs it.